src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -177,7 +175,7 @@ Sentry can also use [releases](/product/releases/) to match uploaded source maps
 
 <Note>
 
-We strongly suggest to use the new Debug IDs system for uploading your artifacts since it provides more reliable symbolication and the old release-based system will be faded out in the future.
+We strongly suggest to use the new Debug IDs system for uploading your artifacts since it provides more reliable unminification and the old release-based system will be faded out in the future.
```

Comment:
```suggestion
We strongly suggest that you use the new Debug ID system for uploading your artifacts since it provides more reliable unminification and the old release-based system will be faded out in the future.
```